src/components/auth/dataUpload.js

```diff
@@ -0,0 +1,143 @@
+import React, {useState} from 'react';
+import { axiosWithAuth } from "../../utils/axiosWithAuth.js";
+import { ifDev } from "../../utils/removeAttribute.js";
+
+import IMDbImg from '../auth/images 2.png'
+import LetterboxdImg from '../auth/Letterboxd_logo_(2018) 3.png'
+import './dataUpload.scss'
+
+const DataUpload = props => {
+
+
+const [input, setInput] = useState({file:''})
+    const [ratings, setRatings] = useState({})
+    const [loading, setLoading] = useState()
+
+  
+
+
+function clikL() {
+    document.getElementById("myDropdownL").classList.toggle("show");
+  }
+
+function clikR() {
+    document.getElementById("myDropdownR").classList.toggle("show");
+  }  
+
+  window.onclick = function(event) {
+    if (!event.target.matches('.dropbtn')) {
+      var dropdowns = document.getElementsByClassName("dropdown-content");
+      var i;
+      for (i = 0; i < dropdowns.length; i++) {
+        var openDropdown = dropdowns[i];
+        if (openDropdown.classList.contains('show')) {
+          openDropdown.classList.remove('show');
+        }
+      }
+    }
+  }
+  let { userid } = props.match.params;
+    const handleChange = e => { 
+        let data = new FormData();
+        data.append("movies", e.target.files[0], e.target.files[0].name);
+        // history.location.state.userid is just where I am holding userid for now from the Register page so I do not need to implment redux.
+        axiosWithAuth()
+          // this is insantiated when a file is added to input
+          .post(`/${userid}/uploading`, data, {
+            headers: {
+              "Content-Type": "multipart/form-data"
+            }
+          })
+          .then(res => {
+            // waiting to set ratings var for 45 seconds.
+            setTimeout(() => setRatings(res.data), 60 * 1000);
```

Comment:
Cody got rid of this time out and just set the ratings once we've recieved a response from the API, but since this was in a different file this was never updated.